account/model.go

```diff
@@ -36,9 +36,17 @@ type Token struct {
 
 // BJJWallet BJJ Wallet
 type BJJWallet struct {
-	PrivateKey    babyjub.PrivateKey
-	PublicKey     babyjub.PublicKeyComp
-	HezBjjAddress string
-	EthAccount    accounts.Account
-	HezEthAddress string
+	PrivateKey      babyjub.PrivateKey
+	PublicKey       babyjub.PublicKeyComp
+	HezBjjAddress   string
+	EthAccount      accounts.Account
+	HezEthAddress   string
+	HezEthSignature string
+}
+
+// AccountCreation is used to submit new account creation to a Hermez Node
+type AccountCreation struct {
```

Comment:
IMO it would be better to add JSON marshaler into `common.AccountCreationAuth` and avoid having an extyra struct just for the marshaling

Reply:
I appreciate your suggestion and it makes sense. It would require changes in Hermez Node and we can open an issue there to implement it later. For now, for the sake of SDK release I believe we can stay with this implementation and we can refactor it later.